code/src/pwm.cpp

```diff
@@ -154,13 +154,15 @@ namespace R2D2::pwm_lib {
         }
     }
     
-    void pwm_c::set_polarity(polarity new_polartiy) {
-        ch_polarity = new_polartiy;
-        if (ch_polarity == polarity::NEGATIVE){
-            PWM->PWM_CH_NUM[ch_nr].PWM_CMR |= PWM_CMR_CPOL;
-        } else {
-            PWM->PWM_CH_NUM[ch_nr].PWM_CMR &= ~PWM_CMR_CPOL;
-        }
+    void pwm_c::set_polarity(polarity new_polarity) {
+	if (new_polarity != ch_polarity){
```

Comment:
I like it more if the happy path is left alligned.

```



```

Reply:
I personally couldn't care less how the if statement is implemented so I changed it to your preference.